src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapperTest.java

```diff
@@ -129,6 +154,30 @@ void testMapPathologyRecordToBundleWithPerformer() {
         );
     }
 
+    @Test
+    void testMapPathologyRecordToBundleWithPerformingOrganization() {
+        final var mockPerformingOrganization = mock(Organization.class);
+        pathologyRecordBuilder.requestingOrganization(mockPerformingOrganization);
+
+        final var bundle = bundleMapper.mapToBundle(pathologyRecordBuilder.build());
+
+        final var performingOrganizationBundleEntries = bundle.getEntry().stream()
+            .filter(entry -> entry.getResource() instanceof Organization)
+            .collect(Collectors.toList());
+        final var practitioners = performingOrganizationBundleEntries.stream()
```

Comment:
Could we rename this to `organizations` to reflect what it contains?
```suggestion
        final var organizations = performingOrganizationBundleEntries.stream()
```

Reply:
Could also also do this in the `testMapPathologyRecordToBundleWithRequestingOrganization` test as well?

Reply:
Oops, copy+paste issue. Fixed now